packages/server/src/seeder/utils.ts

```diff
@@ -313,4 +333,33 @@ export async function addLikesToArticles(_min: number, _max: number): Promise<vo
     // 5. first half of articles will be like by a user
     // 6. remaining half of articles will be dislike by a user
     // Perform the above task in parallel ( Promise.all() ) for each and every user
+
+    const prisma = PrismaClientSingleton.prisma;
+
+    const [articles, users] = await Promise.all([await prisma.article.findMany(), await prisma.user.findMany()])
```

Comment:
There is an issue with the unnecessary use of await inside the Promise.all function. You can simplify the code as follows:

```js 
const [articles, users] = await Promise.all([prisma.article.findMany(), prisma.user.findMany()]);
```